runner-setup/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Dockerfile to setup a new runner for CI service.
+# This file has all dependencies for running the following service(s):
+#   - gen_optimized.py
+
+# Use a lightweight base image
+FROM ubuntu:24.04
+
+# Set environment variables
+# Should probably use docker compose for this
+ENV RUNNER_USER=runner
+ENV RUNNER_HOME=/home/${RUNNER_USER}
+# Should probably use docker compose and to set these values:
+ENV RUNNER_VERSION=2.319.1
+ENV GITHUB_TOKEN=#<Github Token Here>
```

Comment:
Use `RUN --mount=type=secret` for secrets: https://docs.docker.com/reference/dockerfile/#example-mount-as-environment-variable

Reply:
I think just `ARG` will work here.

Reply:
Did secret